python/pyspark/ml/clustering.py

```diff
@@ -17,9 +17,11 @@
 
 from pyspark import since, keyword_only
 from pyspark.ml.util import *
-from pyspark.ml.wrapper import JavaEstimator, JavaModel
+from pyspark.ml.wrapper import JavaEstimator, JavaModel, JavaWrapper
 from pyspark.ml.param.shared import *
 from pyspark.ml.common import inherit_doc
+from pyspark.rdd import ignore_unicode_prefix
+
```

Comment:
So in the other random files I sampled in this directory we only have one new line, was there a reason for this change?